kuzzleio/documentation
# Fix sidebar

## What does this PR do?

The sidebar was designed to handle the redirection from sdk  method page to api same method. 
We removed that possibility and now the redirection only push on the home page of sdk/api.
The sidebar haven't been change in that way.

This pr remove that from sidebar too and allow user to automaticaly open the current page concerned submenu in sidebar. 

### How should this be manually tested?
  - Step 1 : refresh a page of an sdk method, concerned menu should be open
  - Step 2 : Use sdk selector should redirect to home page
  - Step 3 : Home page button (Kuzzle logo), should redirect on home page site

### Other changes
Add deploy how to in travis
### Boyscout
/
